docs/eventing/channels/channel-types-defaults.md

```diff
@@ -5,14 +5,35 @@ type: "docs"
 showlandingtoc: "true"
 ---
 
-Knative provides the InMemoryChannel channel implementation by default. This default implementation is useful for developers who do not want to configure a specific implementation type, such as Apache Kafka or NATSS channels.
+Knative uses two types of channels:
+* The generic Channel object
+* Channel implementations that each have their own custom resource definitions (CRDs), such as
+`InMemoryChannel` and `KafkaChannel`.
 
-**NOTE:** InMemoryChannel channels should not be used in production environments.
+The custom Channel implementations each have their own event delivery mechanisms, such as in-memory
+or broker-based. Example brokers include Kafka and GCP Pub/Sub.
 
-The default channel implementation is specified in the `default-ch-webhook` ConfigMap in the `knative-eventing` namespace.
-For more information about modifying ConfigMaps, see [Configuring the Eventing Operator custom resource](./docs/install/configuring-eventing-cr/).
+Knative provides the `InMemoryChannel` channel implementation by default.
+This default implementation is useful for developers who do not want to configure a specific
+implementation type, such as Apache Kafka or NATSS channels.
 
-In the following example, the cluster default channel implementation is InMemoryChannel, while the namespace default channel implementation for the `example-namespace` is KafkaChannel.
+Through the generic Channel object you can create a channel without specifying which channel
+implementation CRD is used.
+This is useful if you do not care about the properties a particular channel provides, such as
+ordering and persistence, and you want to use the implementation selected by the operator.
+The operator controls the default settings through the `default-ch-webhook` ConfigMap in the
+`knative-eventing` namespace.
+
+For more information about modifying ConfigMaps, see
+[Configuring the Eventing Operator custom resource](./docs/install/configuring-eventing-cr/).
```

Comment:
Is this link correct, I don't see it here:
https://github.com/knative/docs/tree/main/docs/install

But, I do see it here:
https://github.com/knative/docs/tree/main/docs/install/operator
https://github.com/knative/docs/blob/main/docs/install/operator/configuring-eventing-cr.md

Though I'm a bit curious if we need to rejigger that page, since you can modify the configmaps even without the operator, and linking from here might make somebody think that they can only do this if they install operator, or that they have to install the operator.

Reply:
Yeah I think we can remove that paragraph, probably it's safe to assume a user will know how to change the config maps